lib/ui/widgets/login/login_terms_conditions.dart

```diff
@@ -5,6 +5,10 @@ import 'package:url_launcher/url_launcher.dart';
 
 /// Allows to view Mooncake terms and conditions
 class LoginTermsAndConditions extends StatelessWidget {
+  final String align;
```

Comment:
What about using a `TextAlign` property directly and setting it as `TextAlight.start` by default instead of a String here?
Do you think that's possible?